### PR TITLE
Update ExpandedLJ note

### PR DESCRIPTION
**Pull request by cbkerr**
## Description

Updates note about ExpandedLJ
Discussed with @syjlee 

Agree with offline discussion with @joaander that we shouldn't have to add cases to all potentials.
## How has this been tested?

Built locally and [see for yourself here](https://hoomd-blue--1501.org.readthedocs.build/en/1501/module-md-pair.html#hoomd.md.pair.ExpandedLJ)

## Change log

```
Clarify note on how ExpandedLJ handles `r_{cut}`
```

## Checklist:

- [x] I have reviewed the [**Contributor Guidelines**](https://github.com/glotzerlab/hoomd-blue/blob/trunk-minor/CONTRIBUTING.md).
- [x] I agree with the terms of the [**HOOMD-blue Contributor Agreement**](https://github.com/glotzerlab/hoomd-blue/blob/trunk-minor/ContributorAgreement.md).
- [x] My name is on the list of contributors (`sphinx-doc/credits.rst`) in the pull request source branch.

### CODE REVIEW

File: hoomd/md/pair/pair.py [hunk 1]
```diff
@@ -298,8 +298,8 @@ class ExpandedLJ(Pair):
                \right)^{6} \right]
 
     Note:
-        To replicate the behavior of the SLJ potential in HOOMD-blue v2, set
-        `hoomd.md.pair.Pair.r_cut` to ``r_cut_unshifted + delta``.
+        To replicate the behavior of the SLJ potential in HOOMD-blue v2, set the
+        `hoomd.md.pair.Pair.r_cut` to ``r_cut(for delta=0) + delta``.
```
Comment (tommy-waltmann) [inline hoomd/md/pair/pair.py]: This note may be more useful as an entry in the v3 migration guide rather than in the class docstring.

Comment (seoulfood) [reply hoomd/md/pair/pair.py]: Agree with this note.

**Review by shihkual — COMMENTED** [review]
LGTM! Thanks

File: hoomd/md/pair/pair.py (hunk 1 above)
Comment (joaander) [reply hoomd/md/pair/pair.py]: @cbkerr Shall we move it to the migration guide then?

File: hoomd/md/pair/pair.py [hunk 2]
```diff
@@ -298,8 +298,8 @@ class ExpandedLJ(Pair):
                \right)^{6} \right]
 
     Note:
```
Comment (cbkerr) [inline hoomd/md/pair/pair.py]:
```suggestion
```

File: hoomd/md/pair/pair.py [hunk 3]
```diff
@@ -298,8 +298,8 @@ class ExpandedLJ(Pair):
...
-        To replicate the behavior of the SLJ potential in HOOMD-blue v2, set
-        `hoomd.md.pair.Pair.r_cut` to ``r_cut_unshifted + delta``.
+        To replicate the behavior of the SLJ potential in HOOMD-blue v2, set the
+        `hoomd.md.pair.Pair.r_cut` to ``r_cut(for delta=0) + delta``.
```
Comment (cbkerr) [inline hoomd/md/pair/pair.py]:
```suggestion
```

File: sphinx-doc/migrating.rst [hunk 4]
```diff
@@ -157,7 +157,7 @@ HOOMD v3 removes old APIs, unused functionality, and features better served by o
    * - ``f_list`` and ``t_list`` parameters to ``md.force.active``
      - Per-type ``active_force`` and ``active_torque``
    * - ``md.pair.SLJ``
-     - `md.pair.ExpandedLJ`
+     - `md.pair.ExpandedLJ` with `hoomd.md.pair.Pair.r_cut` set to ``r_cut(for delta=0) + delta``.
```
Comment (cbkerr) [inline sphinx-doc/migrating.rst]:
```suggestion
     - `md.pair.ExpandedLJ` with `hoomd.md.pair.Pair.r_cut` set to ``r_cut(for delta=0) + delta``
```

**Review by joaander — APPROVED** [review]
Thanks!

**Pull request merged**